security/cmd/istio_ca/main.go

```diff
@@ -141,6 +142,12 @@ func runCA() {
 	stopCh := make(chan struct{})
 	sc.Run(stopCh)
 
+	serviceController := controller.NewServiceController(cs.CoreV1(), opts.namespace,
```

Comment:
Can we add some comments here to explain what it does? And if possible, please do the similar thing to the above NewSecretController as well, thanks!

Reply:
Will do!

Reply:
The docs are at the struct definition for both controllers.